packages/cycle-scripts-one-fits-all/scripts/init.js

```diff
@@ -84,6 +86,7 @@ module.exports = function init (appPath, appName, verbose, originalDirectory) {
         'start': 'cycle-scripts start',
         'test': 'cycle-scripts test',
         'build': 'cycle-scripts build',
+        'build:clean': 'rimraf build .tmp .nyc_output coverage',
```

Comment:
Change to `cycle-scripts clean` and move the command there.
Don't forget to adjust the eject script